### PR TITLE
Feature authentication

### PR DESCRIPTION
**Pull request by NeHerkus**
Add authentication and login post

### CODE REVIEW

File: src/app/services/authentication.service.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import {BehaviorSubject, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+
+import {User} from '../models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+  baseUrl = 'https://good-deed.herokuapp.com';
```
Comment (gpiliponyte) [inline src/app/services/authentication.service.ts]: should be moved to constant outside the class

File: src/app/homepage/home-page.component.html [hunk 2]
```diff
@@ -12,86 +11,89 @@ <h1><b>Welcome to Summer of Good Deeds! </b></h1>
...
   </div>
-  <br>
-
-</div>
 </div>
-<!--<footer class="w3-container w3-dark-grey w3-padding-32 w3-margin-top">-->
-  <!--<p><a href="https://www.w3schools.com/w3css/default.asp" target="_blank"></a></p>-->
-<!--</footer>-->
+<footer class="w3-container w3-dark-grey w3-padding-32 w3-margin-top">
+  <p><a href="https://www.w3schools.com/w3css/default.asp" target="_blank"></a></p>
```
Comment (gpiliponyte) [inline src/app/homepage/home-page.component.html]: why do you need a link to w3school tutorial

File: src/app/services/authentication.service.ts (hunk 1 above)
Comment (NeHerkus) [reply src/app/services/authentication.service.ts]: Done

File: src/app/homepage/home-page.component.html (hunk 2 above)
Comment (NeHerkus) [reply src/app/homepage/home-page.component.html]: Čia net nežinau ar man tvarkyt, ar ne, nes čia nuo kito stable barancho rebasinta ir liko šita nesąmonė

**Pull request merged**